### DISCUSSION TITLE
[Script request] IT-Tools

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [community-scripts/ProxmoxVE#33] [Request script] [answered]
### Application Name

IT-Tools

### Website

https://github.com/CorentinTh/it-tools

### Description

Useful tools for developer and people working in IT. [Have a look !](https://it-tools.tech/).

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.

**Comment by cwmoriarty** [comment]
Just use the demo site or https://gchq.github.io/CyberChef/? I don't understand why you'd need to self-host that.

**Reply by newzealandpaul** [reply]
I can see the point of hosting, it has password generators etc. a person or org may prefer to run a version whose code they can inspect and run themselves.

**Reply by CrazyWolf13** [reply]
@cwmoriarty First of all IT-Tools is different, and if I use a tool related with my password/my hashes passwords, it definitely has to be selfhosted.

**Reply by GCustom** [reply]
I love this one, would support it, I've got it in docker at the moment

**Comment by newzealandpaul** [comment]
Sorry this has taken a while, lots going on with organising the repo. I will get onto this soon.

**Comment by MickLesk** [comment, marked as answer]
done
